unit_test/unit.cpp

```diff
@@ -0,0 +1,38 @@
+#include <map>
```

Comment:
A teszt kedvéért nem kell másolatokat készíteni a fájlokról.

Reply:
Itt ezeket 2 ok miatt másoltam le. A CMake az unit.cpp fájlt nem találta meg ha úgy hivatkoztam rá, hogy "../unit.cpp", és a jsonparser.h pedig tartalmazott egy "exit -1" parancsot valamelyik hiba esetén ami nem tetszett a googletest-nek. Megpróbálom feltölteni hátha github környezetben jobban lefut.

Reply:
Igen, a Cmake továbra sem találja githubon sem az unit.cpp fájlt és ezért elhasal az unit teszt is. Esetleg itt egy tippet adni, mi lehet rossz?